schemas/expconf/v0/resources.json

```diff
@@ -80,6 +80,13 @@
             "minimum": 0,
             "default": 1
         },
+        "is_single_node": {
+            "type": [
+                "boolean",
+                "null"
+            ],
+            "default": false
+        },
```

Comment:
Why do we need this new config option for generic tasks?

Reply:
The purpose is to give control over scheduling as a single node/pod/container like we schedule commands today vs splitting across multiple (like we used to do for experiments). I've added this new option for expconf in https://github.com/determined-ai/determined/pull/8743 where it's actually necessary. these schema changes will leave this PR on rebase.

Reply:
I see.  The PR description does not indicate it is currently rebased on something else.